### ISSUE TITLE
False positive ramda/cond-simplification

### ISSUE THREAD

**Issue by anion155** [ramda/eslint-plugin-ramda#32]
```javascript
const entityTypeMapper = (...pairs) =>
  R.cond([
    ...pairs.map(([from, to]) => [R.propEq('type', from), R.assoc('type', to)]),
    [R.T, R.identity],
  ]);
```

ESLint config extended from `plugin:ramda/recommended`

**Comment by CrossEye** [comment]
I know nothing about the eslint plugin, so I can't speak to why this generates a warning, but I do want to say that this is a slick piece of code!
